Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackIntegration.m

```diff
@@ -0,0 +1,14 @@
+#import "SentryUserFeedbackIntegration.h"
+#import "SentryOptions+Private.h"
+#import "SentrySwift.h"
+
+@implementation SentryUserFeedbackIntegration {
```

Comment:
l: Do you plan to add more features to this class?
Why not use `SentryUserFeedbackIntegrationDriver` as the integration like before?

Reply:
Originally I had it like that, but it wasn't compiling with a swift integration class, and we don't have any others like that. I essentially tried to copy what you have for replays.